examples/video_modes.rs

```diff
@@ -3,7 +3,13 @@ use winit::event_loop::EventLoop;
 fn main() {
     simple_logger::init().unwrap();
     let event_loop = EventLoop::new();
-    let monitor = event_loop.primary_monitor();
+    let monitor = match event_loop.primary_monitor() {
+        Some(monitor) => monitor,
+        None => {
+            println!("No primary monitor detected.");
+            return;
+        }
+    };
```

Comment:
Probably easier to just unwrap/expect here?

Reply:
Maybe, but you can run this on Wayland without any issue and got your `no primary monitor detected`, since this example doesn't require window in particular, I don't really like panic, since the intention is to show that it's **normal** that primary monitor isn't available.

Reply:
Sure, certainly not unreasonable.